bench/workgen/runner/example_prepare.py

```diff
@@ -52,21 +52,21 @@
 assert ret == 0, ret
 
 opread = Operation(Operation.OP_SEARCH, table)
-read_txn = txn(opread * 5, 'read_timestamp')
+read_txn = op_group_transaction(opread, 5, "read_timestamp")
```

Comment:
While this may fix the problem, I wonder if this reveals something to be broken in the `txn()` function?  Is it possible to fix `txn` so it works with the original code?  `txn` was meant to be used by hand coded programs such as this one, whereas `op_group_transactions` was meant to be used by wtperf.py to emulate closely some aspect of how wtperf groups operations in a transaction.

Reply:
My understanding is that `op_group_transactions` is to create transactions around groups of operations of the indicated size. The comment [here](https://github.com/wiredtiger/wiredtiger/blob/develop/bench/workgen/runner/runner/core.py#L347) in core.py also mentions the same thing.

Reply:
I looked this some more -- I believe `op_group_transactions` does the following:  given a list of operations, it groups them into up to N in a batch, and wraps a transaction around that batch, doing that for as many batches as it takes.

Given that, `op_group_transaction(opread, 5, "read_timestamp")` takes `opread`, which is just one operation, and wraps a transaction around it.  So it's not the same thing as `txn(opread * 5, 'read_timestamp')` at all.

@raviprakashgiri29, can you try this code with first:
```
read_txn = txn(opread, 'read_timestamp')
```
making that same change for `write_txn` and `update_txn`.  I suspect that will work, and is the same as what you have currently with `op_group_transaction`.

Then, can you try:
```
read_txn = op_group_transaction(opread * 5, 5, "read_timestamp")
```
making that same change for `write_txn` and `update_txn`.  I suspect that will fail, and is the same as what was originally there with `txn(opread * 5, 'read_timestamp')`.

If that theory is correct, then I think you should use `read_txn = txn(opread, 'read_timestamp')`, as it should "solve" the problem, in a more natural way.  And if that is true, I think what may be going on with update, is that sometimes update may be selecting the same key to work on in a transaction.  If there are five updates in a transaction, and two of the updates change key=113, that may be illegal to do in a single transaction.  Putting just a single update in a transaction will mean that can't ever happen.

Reply:
Thanks @ddanderson for your suggestions. I tried this before to replace `txn(opread * 5, 'read_timestamp')` with `txn(opread, 'read_timestamp')` (and other transactions) and it works. Also, as you said it is more of a natural way to fix.

Reply:
Fixed 9093022fdb72ea54097faca7207435b3c656c5a0